app/javascript/components/visual-settings-form/expression-editor/expression-editor-style.css

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
put every style changes inside a common classname so that it won't effect existing ones by any chance. like - 

```
.miq-expression-container {
  .edit-expression-container {
....
...
...
}
```